docs/src/embeddings/custom_embedding_function.md

```diff
@@ -55,6 +55,14 @@ Let's implement `SentenceTransformerEmbeddings` class. All you need to do is imp
 
 This is a stripped down version of our implementation of `SentenceTransformerEmbeddings` that removes certain optimizations and default settings.
 
+!!! danger "Use sensitive keys to prevent leaking secrets"
+    To prevent leaking secrets, such as API keys, you should add an configuration
+    parameteres of embedding function to the output of the
```

Comment:
```suggestion
    To prevent leaking secrets, such as API keys, you should add any sensitive
    parameters of an embedding function to the output of the
```